app/views/design-system/rsd-design-system/design-patterns/index.html

```diff
@@ -0,0 +1,64 @@
+{% extends "layouts/content-with-sidenav.html" %}
+
+{% set pageName = "Our approach" %}
+{% set pageDescription = "How we design and build services at the Regional Services Division." %}
+{% set selectedNav = "Design system" %}
+{% set setSelectedSideNav = "Our approach" %}
+{% set parentSection = "rsd-design-system"%}
+{% set isChildSection = "true"%}
+{% set isSubNavSection = "true" %}
+{% set lastUpdated = "June 2024 "%}
+{% set backlog_issue_id = "" %}
+
+{% block beforeContent %}{% endblock %}
+
+{% block content %}
+<p>In Regional Services Division (RSD), we use the GOV.UK Design System so that the look and function of the products is similar across the suite, and
+    to GOV.UK. </p>
+<p>However, each team has interpreted and applied the design system to their context. This means that the products may
+    be meeting similar or the same needs, but the solutions look or function differently.</p>
+<p>Over time we want our suite of products to become consistent, so that it&#39;s easier for users to learn and use
+    multiple products. </p>
+<p>By creating a library of design patterns and applying these to meet user needs, we will create this similarity and
+    also:</p>
+<ul>
+    <li>it will be quicker to design and build our products</li>
+    <li>we will spend less time solving problems which other teams have already solved</li>
+    <li>it will make it easier to maintain the suite of products once they are managed by Service Support</li>
+</ul>
+<p>We have had some user feedback that it&#39;s easy to use a product because they are familiar with another product
+    which works similarly. We have also had feedback that where one product does something but another product does not,
+    it&#39;s confusing and they waste time trying to work out how to achieve their goal. </p>
+<p>We will continue to gather insight from users to understand where inconsistency is causing them problems, and where
+    consistency may help them. </p>
+<h2>Rolling-out design patterns across RSD</h2>
+<h3>When designing something new or improving an existing design</h3>
+<p>We check the <a href="https://educationgovuk.sharepoint.com/sites/ServiceDeliveryDirectorate/Lists/RSD%20crossprogramme%20user%20needs/AllItems.aspx?env=WebViewList">user needs database</a> to see if a user need we have identified is
```

Comment:
Internal link again

Reply:
In the updated version, this part is now:

`<a href="https://educationgovuk.sharepoint.com/sites/ServiceDeliveryDirectorate/Lists/RSD%20crossprogramme%20user%20needs/AllItems.aspx?env=WebViewList">user needs database (opens on DfE intranet)</a>`